INSTALL.md

```diff
@@ -53,16 +53,18 @@ You will need:
  * GLEW
  * libpng
  * gettext
+ * librsvg2
```

Comment:
Neither librsvg2 nor xmlstarlet are required for building Colobot. They are only required for generating desktop icons. They are completely optional and so definitely shouldn't be listed under "you need this" list.